src/normalize/entities.ts

```diff
@@ -349,6 +350,11 @@ const ForwardMessageInput = new Entity({
 	su: 'subject'
 });
 
+export const ForwardAppointmentInfo = new Entity({
+	m: ['message', ForwardMessageInput],
+	exceptId: ['exceptId', InstanceDate]
```

Comment:
`exceptId` is not required to be included here.

Reply:
not sure I understood. We have to pass exceptId when we want to forward an instance right?

Reply:
As discussed with you.. this is needed hence we need to keep it.

Reply:
Sorry. Missed above `InstanceDate`. Thanks @craigdanj